pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[project]
+name = "gcp-vm-control"
+version = "0.1.0"
+description = "Add your description here"
```

Comment:
Add a description for the project here, something short about what gcp-vm-control is.